### PR TITLE
Feature auth service

### PR DESCRIPTION
**Pull request by FrozenPear42**

### CODE REVIEW

File: .github/workflows/publish.yml [hunk 1]
```diff
@@ -17,6 +17,8 @@ jobs:
         run: |
           export CONTAINER_LOWERCASE="${CONTAINER_IMAGE,,}"
           echo $CONTAINER_LOWERCASE
+          echo ${{secrets.privk}} > ./keys/jwrRS256.key
+          echo ${{secrets.pubk}} > ./keys/jwrRS256.key.pub
```
Comment (RobPiwowarek) [inline .github/workflows/publish.yml:17]: very good professional change

File: app/resources.py [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+            'password', help='This field cannot be blank', required=True)
+        self.parser.add_argument(
+            'email', help='This field cannot be blank', required=True)
+
+    def post(self):
+        data = self.parser.parse_args()
+        username = data['username']
+        password = data['password']
+        email = data['email']
+        password_hash = bcrypt.hashpw(
```
Comment (RobPiwowarek) [inline app/resources.py]: why do this new line here

File: keys/jwtRS256.key [hunk 3]
```diff
@@ -0,0 +1,51 @@
+-----BEGIN RSA PRIVATE KEY-----
```
Comment (RobPiwowarek) [inline keys/jwtRS256.key]: we dont need this keys anymore since we added them as secrets right?

**Review by RobPiwowarek — CHANGES_REQUESTED** [review]

File: keys/jwtRS256.key (hunk 3 above)
Comment (FrozenPear42) [reply keys/jwtRS256.key]: those are included as develop/debug keys

File: .github/workflows/publish.yml (hunk 1 above)
Comment (Grushenko) [reply .github/workflows/publish.yml:21]: great change my boi

File: app/resources.py (hunk 2 above)
Comment (FrozenPear42) [reply app/resources.py]: linter rules for max line length

**Review by RobPiwowarek — APPROVED** [review]

**Pull request merged**